include/acl_hal_mmd.h

```diff
@@ -149,6 +149,9 @@ typedef struct {
   // accounted and returned in a subsequent invocation of this function.
   void (*aocl_mmd_simulation_streaming_kernel_done)(
       int handle, const std::string &signal_name, unsigned int &finish_counter);
+
```

Comment:
Hi Sophie, can you please add some comments to describe this mmd function? Thanks!